pkgs/top-level/python-packages.nix

```diff
@@ -12068,6 +12068,11 @@ self: super: with self; {
 
   protobuf3-to-dict = callPackage ../development/python-modules/protobuf3-to-dict { };
 
+  # Protobuf 3.x
+  protobuf3 = callPackage ../development/python-modules/protobuf/3.nix {
```

Comment:
I think this should go above protobuf3-to-dict, to fix the sorting. Probably.

Reply:
Good catch, fixed now.